chisel-wrapper

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# USAGE:
+#   ./<script-name> [ chisel-cut-OPTIONS ] <slices..>
+# Installs the slices, using
+#   chisel cut "$@"
+# Generates a status file similar to /var/lib/dpkg/status at the CHISEL_DPKG_STATUS_FILE path
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_CACHE_DIR          (optional) the path which chisel will use to store it's cache
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -e
+
+CHISEL_BIN="${CHISEL_BIN:-"./chisel"}"
+CHISEL_CACHE_DIR="${CHISEL_CACHE_DIR:-".chisel-cache"}"
+CHISEL_DPKG_STATUS_FILE="${CHISEL_DPKG_STATUS_FILE}"
+
+_print_error() {
+	echo "Error: $@" >> /dev/stderr
```

Comment:
nitpick: the indentation looks weird...did you run `shellcheck` on this?

Reply:
Right, thanks for the reminder. Indentation was okay, but `shellcheck` did pick up something else on this line. Fixed it.